content/chainguard/chainguard-registry/pull-through-guides/artifactory-pull-through/index.md

```diff
@@ -137,13 +137,13 @@ Be sure to include your own username and Artifactory instance.
 
 After running this command, you'll be prompted to enter a password. Copy the token from the second code block, paste it into your terminal, and press `ENTER`.
 
-After running the `docker login` command, you will be able to pull a Chainguard Production Image through Artifactory. The following example pulls the `wolfi-base` Image.
+After running the `docker login` command, you will be able to pull a Chainguard Production Image through Artifactory. The following example will pull the `argo-cli-fips` Image if your organization has access to it:
```

Comment:
I wonder if chainguard-base would be a better choice (more common and clearer what it is), but it doesn't make much difference.

Reply:
I like that idea, will change this